app/helpers/payment_details_helper.rb

```diff
@@ -1,4 +1,7 @@
+# frozen_string_literal: true
+
 module PaymentDetailsHelper
+  delegate :billing_address, to: :current_account
```

Comment:
This should be `private`

Reply:
You mean adding private modifier, like 
`private delegate :billing_address, to: :current_account`

Or adding a private section at the end of file, like
```
private
delegate :billing_address, to: :current_account
```
?

Reply:
Or maybe it's better to go back to previous approach?
`billing_address = current_account.billing_address`
WDYT @Martouta ?

Reply:
The previous approach looks good enough to me